docs/source-control/new-repo.md

```diff
@@ -0,0 +1,46 @@
+---
+id: new-repo
+slug: /source-control/new-repo
+title: New Repository
+
+---
+
+## Basic Setup
+
+* Initialize the repository with `git init` (cli)
+* Create a new repository on github (github)
+* Select The Art of Education as the owner (github)
+* Select the proper repo privacy (github)
+
+![Github New Repo](github-new-repo.png)
+
+* Set the origin with `git remote add origin git@github.com:theartofeducation/your-repo-here.git` (cli)
+* create your initial commit and push (cli)
+
+## Common Linting Dependencies
+
+* All projects will require common linting dependencies and these are all written in JavaScript
+    1. [commitlint](https://github.com/conventional-changelog/commitlint)
+    1. [markdownlint](https://github.com/DavidAnson/markdownlint)
+    1. [husky](https://github.com/typicode/husky)
+* All projects will have a `package.json` for installing these dependencies and scripts to run these tasks as needed.
+* All projects will have a github actions workflow for running these tasks i.e. `.github/workflows/linting.yml`
+
+## Set Default Branch
+
+Some Repo ( such as services and frontends ) the default branch will be `develop` as we need
+to have separate deployable units for acceptance and production environments. others ( like libraries
+and plugins ) don't need this and the default branch will be `main`.
+
+## Branch Rules & Setup
```

Comment:
🤔 When setting up the branch rules, what is the guidance for the "Branch name pattern"?
<img width="1703" alt="Screen Shot 2021-09-07 at 12 56 46 PM" src="https://user-images.githubusercontent.com/12145163/132382987-4bbeccbd-a636-49f0-a30d-894d7d8761ba.png">

Reply:
`main` or `develop`